test/multiconductor.jl

```diff
@@ -225,6 +225,21 @@ end
             end
         end
 
+
+        @testset "iv 5-bus case" begin
+            result = PowerModels._run_mc_opf_iv(mp_data, IVRPowerModel, ipopt_solver)
+
+            @test result["termination_status"] == LOCALLY_SOLVED
+            #TODO uncomment
```

Comment:
Should this part of the test be restored?

Reply:
partially restored, as it gives different results from ACP (not sure this test gives unique solutions?)

Reply:
As the IV formulation is non-convex, it is possible that Ipopt will converge to a different point than the other non-convex formulations.  When I implemented IV back in 2014 I found that it at had the same objective value in all cases as ACP/ACR.  So as long as that value is matching that seems to be consistent with past experiments.

Reply:
problem found and solved. Found some inconsistencies with how build_mc_data! builds the data model vs how PowerModelsDistribution expects it. Will open an issue